components/file_preview_modal/file_preview_modal.tsx

```diff
@@ -336,7 +336,7 @@ export default class FilePreviewModal extends React.PureComponent<Props, State>
                     dialogClassName += ' modal-code';
                     content = (
                         <CodePreview
-                            fileInfo={fileInfo}
+                            fileInfo={fileInfo as FileInfo}
```

Comment:
does the type not match here?

Reply:
Yes, the type for fileInfo is **FileInfo | LinkInfo**. And without this type type mismatch error **Type 'LinkInfo' is missing the following properties from type 'FileInfo': id, user_id, create_at, update_at, and 7 more** comes up

Reply:
> Yes, the type for fileInfo is **FileInfo | LinkInfo**. And without this type, mismatch error "**Type 'LinkInfo' is missing the following properties from type 'FileInfo': id, user_id, create_at, update_at, and 7 more**" comes up